release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -57,7 +57,8 @@ Release Lead is a very time-consuming role, especially towards the end of the re
 
 - Weeks 1-4: 4-8 hours a week
 - Weeks 5-8: 6-12 hours a week
-- Weeks 9-13: 10 to 25 hours a week, including some after-hours and weekend work
+- Weeks 9-13: 10 to 25 hours a week
+- Release Day: Prepare to spend all day on release-related activities.
```

Comment:
Should this be even more explicit, eg: "Given the choice between working after-hours or weekends, and having the release be later than the initial target, delaying the release is preferred."